src/components/ConditionNav/ConditionNav.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import { useParams, NavLink } from 'react-router-dom';
+import i18n, { langTokens } from '../../locales/localizationInit';
+import { useStyles } from './ConditionNav.styles';
+
+export default function ConditionNav(): JSX.Element {
+  const classes = useStyles();
+
+  const { to } = useParams<{ to: string }>();
+
+  useEffect(() => {
+    const element: HTMLElement | null = document.getElementById(to);
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+    element && element.scrollIntoView({ block: 'start', behavior: 'smooth' });
```

Comment:
nit: you may remove the disabling and do this:

`element?.scrollIntoView({ block: 'start', behavior: 'smooth' });`